src/registration/Register.tsx

```diff
@@ -0,0 +1,201 @@
+        <LabelledTextInput
+          required
+          value={givenName}
+          onChange={setGivenName}
+          inputStyle={{ display: 'block' }}
+          label="First Name"
+        />
+        <LabelledTextInput
+          value={familyName}
+          onChange={setFamilyName}
+          inputStyle={{ display: 'block' }}
+          label="Last Name"
+        />
+      </div>
+      <div style={{ lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={email}
+          required
+          onChange={setEmail}
+          labelStyle={{ display: 'block', marginTop: '2rem' }}
+          inputStyle={{ width: '66ex' }}
+          label="Contact Email for Notifications"
+        />
+        <LabelledTextInput
+          value={institute}
+          required={partOfOrganization}
+          disabled={!partOfOrganization}
+          onChange={setInstitute}
+          inputStyle={{ width: '66ex' }}
+          label="Organization"
+        />
+      </div>
+      <div style={{ lineHeight: '170%', marginTop: '0.25rem' }}>
+        <LabeledCheckbox
+          checked={!partOfOrganization}
+          onChange={() => setPartOfOrganization(!partOfOrganization)}
+          disabled={false}
+        >
+          <span style={{ marginLeft: '0.25rem' }}>I am not a part of an organization</span>
+        </LabeledCheckbox>
+      </div>
+      <div style={{ display: 'flex', lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={department}
+          required={partOfOrganization}
+          disabled={!partOfOrganization}
+          onChange={setDepartment}
+          labelStyle={{ display: 'block' }}
+          label="Department"
+        />
+        <LabelledTextInput
+          value={title}
+          required={partOfOrganization}
+          disabled={!partOfOrganization}
+          onChange={setTitle}
+          label="Title"
+          labelStyle={{ display: 'block' }}
+        />
+      </div>
+      <FormLabel style={{ marginTop: '2rem' }}>
+        I am most interested in using Terra to (Check all that apply):
+      </FormLabel>
+      <div style={{ marginRight: '1rem' }}>
+        {[
+          'Collaborate with individuals within my organization',
+          'Collaborate with individuals outside of my organization',
+          'Access data',
+          'Manage datasets',
+          'Launch workflows',
+          'Complete interactive analyses',
+          'Build tools',
+        ].map((title: string) => {
+          return (
+            <InterestInTerraCheckbox
+              key={title}
+              title={title}
+              interestInTerra={interestInTerra}
+              onChange={setInterestInTerra}
+            />
+          );
+        })}
+      </div>
+      <FormLabel style={{ marginTop: '2rem' }}>Communication Preferences</FormLabel>
+      <CommunicationPreferencesCheckbox title="Necessary communications related to platform operations" value />
```

Comment:
Is this supposed to have a value?

Reply:
The linter seems to want this

Reply:
Oh, I see. `value` is always true.

This does work then... passing the prop name alone is equivalent to passing true for the prop.

Changing the name of the value prop to "checked" might make this clearer.